saga/seata-saga-engine-store/src/main/java/io/seata/saga/engine/config/DbStateMachineConfig.java

```diff
@@ -48,6 +48,7 @@
     private String dbType;
     private int transOperationTimeout = DEFAULT_TRANS_OPER_TIMEOUT;
     private SagaTransactionalTemplate sagaTransactionalTemplate;
+    private boolean skipBranchReportOnSuccess;
```

Comment:
https://github.com/seata/seata/blob/a1a51e0c3d374ed0f9604012e9aac1f9822a7495/script/client/spring/application.yml#L9   There is a similar function switch in AT mode. If this configuration can be combined?

Reply:
Done